src/components/skeleton/styles.ts

```diff
@@ -1,7 +1,7 @@
 import { styled } from '@linaria/react'
 
 export const ElSkeleton = styled.span`
-  display: inline-block;
+  display: flex;
```

Comment:
**question:** what was the motivation behind this change?

Reply:
**answer:** I was facing challenges using `display: inline-block` for the parent element, which did not align the child element in the center. So, I updated the `display` property to flex.

FYI: Figma does not have the `display` property, which I used to support the child element alignment in the parent container.